torch/csrc/utils/structseq.cpp

```diff
@@ -0,0 +1,43 @@
+#include "torch/csrc/utils/structseq.h"
+#include "structmember.h"
+#include <sstream>
+
+namespace torch {
+namespace utils {
+
+PyObject *returned_structseq_repr(PyStructSequence *obj) {
+    PyTypeObject *typ = Py_TYPE(obj);
+    std::stringstream ss;
+    ss << typ->tp_name << "(\n";
```

Comment:
The default behavior does not have `\n` here, I added this just because I think it looks better.